packages/schema/src/Schema.ts

```diff
@@ -3778,6 +3778,23 @@ export const clamp =
       { strict: false }
     )
 
+/**
+ * Negates a `Number`.
+ *
+ * @category number transformations
+ * @since 1.0.0
+ */
+export const negate = <A extends number, I, R>(
+  self: Schema<A, I, R>
+): Schema<A, I, R> =>
+  transform(
+    self,
+    typeSchema(self),
+    (self) => N.negate(self),
```

Comment:
this could simply be `N.negate`

Reply:
The `A extends ...` pattern is useful when defining filters, this could simply be:
```ts
export const Negate = S.transform(S.number, S.number, N.negate, N.negate)
```

However, I wonder whether it's worth adding this kind of schemas. There are **many** functions from which you can derive a transformation (`negate`, `Math.abs`, `increment`, etc.), but we can't include all of them.

Reply:
The reason I thought it would be useful in this case is that there is already a bigDecimalNegate and negate is bidireccional negate(negate(a)) = a. But you're right, there are too many fns that could possibly be transformations. I'm not sure where we should draw the line.